apps/full-site-editing/package.json

```diff
@@ -41,8 +41,8 @@
 		"@automattic/calypso-build": "^4.1.0",
 		"@testing-library/jest-dom": "^4.1.0",
 		"@testing-library/react": "^9.1.3",
-		"@wordpress/jest-preset-default": "^5.0.0",
-		"@wordpress/scripts": "^3.4.0",
+		"@wordpress/jest-preset-default": "5.1.1",
+		"@wordpress/scripts": "5.0.1",
```

Comment:
Will need an entry at the changelog

Reply:
Ach, sorry, meant this for `calypso-build`: https://github.com/Automattic/wp-calypso/pull/36583#r334886691

Reply:
should we keep these as caret ranges instead of explicit pins?